custodia/secrets.py

```diff
@@ -174,7 +174,11 @@ def _list(self, trail, request, response):
                 'Content-Type'] = 'application/json; charset=utf-8'
             response['output'] = msg.reply(keylist)
         except CSStoreError:
+            self.logger.error('List function: Internal server error')
```

Comment:
'function' doesn't really sound right.
This is more of an 'operation' failing, so perhaps these should all be '<type> operation', or just the operation type alone: 'List: Internal server error'